lib/ain-cpp-imports/src/bridge.rs

```diff
@@ -50,5 +50,7 @@ pub mod ffi {
         fn getNumCores() -> i32;
         fn getCORSAllowedOrigin() -> String;
         fn getNumConnections() -> i32;
+        fn isDebugEnabled() -> bool;
```

Comment:
Let's retrospectively change this to isEthDebugRPCEnabled, isEthDebugTraceEnabled, etc.

Reply:
Resolved on PR https://github.com/DeFiCh/ain/pull/2641